model/fv_arrays.F90

```diff
@@ -1618,6 +1621,13 @@ subroutine allocate_fv_atmos_type(Atm, isd_in, ied_in, jsd_in, jed_in, is_in, ie
            Atm%phis(i,j) = real_big
         enddo
      enddo
+#ifndef USE_COND
+     do j=jsd, jed
+        do i=isd, ied
+           Atm%q_con(i,j,1) = 0.
+        enddo
+     enddo
+#endif
```

Comment:
Instead of adding another ifndef block here, you can safely put the `Atm%q_con=0`, along with proper documentation,  where it is allocated at line 1555.  In this case, you are not initializing the memory in an OpenMP region.